src/org/thoughtcrime/securesms/ConversationListActivity.java

```diff
@@ -92,8 +92,8 @@ protected void onCreate(Bundle icicle, boolean ready) {
     // update messages - for new messages, do not reuse or modify strings but create new ones.
     // it is not needed to keep all past update messages, however, when deleted, also the strings should be deleted.
     DcMsg msg = new DcMsg(dcContext, DcMsg.DC_MSG_TEXT);
-    msg.setText(getString(R.string.update_1_10_android));
-    dcContext.addDeviceMsg("update_1_10b_android", msg); // addDeviceMessage() makes sure, messages with the same id are not added twice
+    msg.setText(getString(R.string.update_1_10_android) + "\n\n" + "https://delta.chat/en/2020-06-24-releases");
+    dcContext.addDeviceMsg("update_1_10d_android", msg); // addDeviceMessage() makes sure, messages with the same id are not added twice
```

Comment:
As a result of this, those who have installed testing versions will have two messages now? Not a problem for testers, but maybe just keep old message for them.

Reply:
yip this change from 'b' to 'd' is not really needed.